apps/sms/js/thread_ui.js

```diff
@@ -1113,17 +1083,32 @@ var ThreadUI = {
    * @returns {Promise} Promise that is resolved once navigation is completed.
    */
   navigateToComposer: function(parameters) {
```

Comment:
I haven't renamed it yet, but probably we can think of better name for this method now - ```composeNewMessage``` or smth.